tests/python/pytorch/graphbolt/test_base.py

```diff
@@ -248,6 +248,33 @@ def test_isin_non_1D_dim():
         gb.isin(elements, test_elements)
 
 
+def torch_expand_indptr(indptr, dtype, nodes=None):
+    if nodes is None:
+        nodes = torch.arange(len(indptr) - 1, dtype=dtype, device=indptr.device)
+    return nodes.to(dtype).repeat_interleave(indptr.diff())
+
+
+@pytest.mark.parametrize("nodes", [None, True])
+@pytest.mark.parametrize("dtype", [torch.int32, torch.int64])
+def test_expand_indptr(nodes, dtype):
+    num_nodes = 13
+    if nodes:
+        nodes = torch.randint(0, 199, [num_nodes], dtype=dtype, device=F.ctx())
```

Comment:
It is better to design the test cases, rather than leverage random to "luckily" catch bug.

Reply:
In this case, luckily means %99.99999999999, more reliable than a hardcoded small test case. However, I can change it to a hardcoded test case if you insist.